.github/workflows/check-deprications.yaml

```diff
@@ -0,0 +1,34 @@
+name: Check chart compatibility with Kubernetes
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    # Weekly: Monday at 5:30
+    - cron: '0 6 * * 1'
+
+env:
+  K8S_TARGET_VERSION: "v1.25.0"
+
+jobs:
+  check-compatibility:
+    strategy:
+      matrix:
+        kubeVersion: ["v1.23.0", "v1.24.0", "v1.25.0"]
```

Comment:
Are there any plans to add the supported versions to the README?

Reply:
Yes in the CI PR I have updated the README but I still need to work on it

Reply:
There's a compatibility section in the README:

> ### Compatibility
>
> Currently RADAR-Kubernetes is supported on Kubernetes version 1.23 and 1.24, with experimental support on 1.25. You should be able to install the platform on other versions of Kubernetes as well but you might encounter compatibility issues.